distributor-api-v1/operations.md

```diff
@@ -744,9 +744,9 @@ Gives a pricing information for the given configuration.
 | :--- | :--- | :--- | :--- |
 | `OccupancyPrices` | array of [RoomOccupancyAvailability](operations.md#roomoccupancyavailability) | required | Pricing information. |
 
-## Get Payment Gateway   <a id="create-reservation-group"></a>
+## Get Payment Configuration   <a id="get-payment-configuration"></a>
 
-### Request`[PlatformAddress]/api/distributor/v1/payments/getPaymentGateway`   <a id="request-platformaddressapidistributorv1reservationgroupscreate"></a>
+### Request`[PlatformAddress]/api/distributor/v1/hotels/getPaymentConfiguration`   <a id="request-getpaymentconfiguration"></a>
```

Comment:
@dobras Maybe try to put the URL of request to a new line so the URL is not broken to the next line in default browser size? Not sure if it will be better, but I don't like the URL spreading to two lines on a normal window size.